tools/topology/topology2/include/components/multiband_drc.conf

```diff
@@ -0,0 +1,85 @@
+#
+#
+# A generic MULTIBAND_DRC component. All attributes defined herein are namespaced
+# by alsatplg to "Object.Widget.multiband_drc.attribute_name"
+#
+# Usage: this component can be used by declaring in the parent object. i.e.
+#
+# Object.Widget.multiband_drc."N" {
+#		index			1
+#	}
+# }
+
+#
+# Where M is pipeline ID and N is a unique integer in the parent object.
+
+Class.Widget."multiband_drc" {
+	#
+	# Pipeline ID
+	#
+	DefineAttribute."index" {
+		type "integer"
+	}
+
+	#
+	# Unique instance for MULTIBAND_DRC widget
+	#
+	DefineAttribute."instance" {
+		type "integer"
+	}
+
+	# Include common widget attributes definition
+	<include/components/widget-common.conf>
+
+	attributes {
+		!constructor [
+			"index"
+			"instance"
+		]
+		!mandatory [
+			"num_input_pins"
+			"num_output_pins"
+			"num_input_audio_formats"
+			"num_output_audio_formats"
+		]
+
+		!immutable [
+			"uuid"
+			"type"
+		]
+		!deprecated [
+			"preload_count"
+		]
+		unique	"instance"
+	}
+
+	#
+	# multiband_drc widget switch control
+	#
+	Object.Control {
+                mixer."1" {
+                        Object.Base.channel.1 {
+                                name    "fc"
+                                shift	0
+                        }
+                        Object.Base.ops.1 {
+                                name	"ctl"
+                                info	"volsw"
```

Comment:
volsw? why not just switch?

Reply:
I'm not sure if switch would work, I haven't tried. This is also the same as in tplg1 https://github.com/thesofproject/sof/blob/7c013b27e541130bc6b8f3ac6a4a8b22b07cf91a/tools/topology/topology1/sof/pipe-drc-eq-volume-demux-playback.m4#L51.

Switch control is a mixercontrol (volume) with max defined as 1, so it likely needs to be this. I can try what happens with switch.

Reply:
I'm not sure about topology2. But for topology1 there are 3 control tytes: bytes, volsw, enum. Just as Seppo said, there is no specific type for switch, we use volsw type with max value 1 instead